src/main/java/org/isf/ward/rest/WardController.java

```diff
@@ -43,11 +43,12 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
-import io.swagger.annotations.Api;
-import io.swagger.annotations.Authorization;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
-@RestController
-@Api(value = "/wards", produces = MediaType.APPLICATION_JSON_VALUE, authorizations = {@Authorization(value="apiKey")})
+@RestController(value = "/wards")
+@Tag(name = "Wards")
```

Comment:
Please consider to add this to `application.properties.dist`

```
### Swagger-UI (info)
api.host=localhost:8080
api.protocol=http
springdoc.swagger-ui.tagsSorter=alpha <--- for tag sorting in the swagger UI
```

Reply:
okay